boards/linux_process_x86_64/src/main.rs

```diff
@@ -0,0 +1,146 @@
+//! Board file for Linux process for AMD64
+
+// Disable this attribute when documenting, as a workaround for
+// https://github.com/rust-lang/rust/issues/62184.
+// #![cfg_attr(not(doc), no_main)]
+#![feature(const_in_array_repeat_expressions)]
+// #![deny(missing_docs)]
+
+// use capsules::virtual_alarm::VirtualMuxAlarm;
+use kernel::common::dynamic_deferred_call::{DynamicDeferredCall, DynamicDeferredCallClientState};
+use kernel::component::Component;
+use kernel::Platform;
+use kernel::{capabilities, create_capability, debug, static_init};
+use std::panic;
+
+// /// Support routines for debugging I/O.
+pub mod io;
+
+// Number of concurrent processes this platform supports.
+const NUM_PROCS: usize = 4;
+
+// Actual memory for holding the active process structures.
+#[no_mangle]
+static mut PROCESSES: [Option<&'static dyn kernel::procs::ProcessType>; NUM_PROCS] =
+    [None, None, None, None];
+
+static mut CHIP: Option<&'static linux_x86_64::chip::Linux> = None;
+
+// How should the kernel respond when a process faults.
+const FAULT_RESPONSE: kernel::procs::FaultResponse = kernel::procs::FaultResponse::Panic;
+
+#[link_section = ".apps"]
+static mut APP_FLASH: [u8; 256 * 1024] = [0; 256 * 1024];
+static mut APP_RAM: [u8; 256 * 1024] = [0; 256 * 1024];
+
+// #[no_mangle]
+// #[link_section = ".stack_buffer"]
+// pub static mut STACK_MEMORY: [u8; 0x1000] = [0; 0x1000];
+
+struct LinuxProcess {
+    console: &'static capsules::console::Console<'static>,
+    ipc: kernel::ipc::IPC,
```

Comment:
```suggestion
    ipc: kernel::ipc::IPC<NUM_PROCS>,
```

Reply:
Where is this defined? In ipc.rs is does not seem to have any parameter, am I missing something?

Reply:
I think you just need to rebase to master. This is a recent change.

Reply:
It works now, but why did the tests run against the master branch?

Reply:
The CI always runs against the merged result, as that's what's interesting to see for Tock: do we still pass all tests if this is merged, along with all changes that made it into master in the meantime? Moreover, Bors will queue up each change and run the CI prior to a merge again, such that the status _just before the merge_ and not the CI status from the last push on the PR counts.